editor/vim/README.md

```diff
@@ -0,0 +1,10 @@
+## jsonnet-language-server integration for vim
+
+The LSP integration will depend on the vim plugin you're using:
+
+* `mattn/vim-lsp-settings`:
+  * Follow new LSP instalation from <https://github.com/mattn/vim-lsp-settings>
+  * LSP settings file: [jsonnet-language-server.vim](jsonnet-language-server.vim)
+* `neoclide/coc.nvim`:
+  * Inside vim, run: `:CocConfig` (to edit `~/.vim/coc-settings.json`)
+  * Copy [coc-settings.json](coc-settings.json) content
```

Comment:
I'm thinking we want to mention here that the args in `coc-settings.json` and `jsonnet-language-server.vim` are adjustable (i.e. they could adjust it JPATH, tanka enablement, linting diagnostics, etc).

Reply:
+1 improving that README stanza